docs/api/users-update-by-id.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+---
+
+# Update user by ID
+
+Allows the editing of [`user`](user) objects that already exist. You will need to know the `id` parameter of the user you want to edit.
+
+> [!IMPORTANT]
+> The information for the user is replaced **entirely** by the request body, so if you want to keep information the same you still need to include it in the request body.
+
+## URL
+
+```shell
+{server_url}/users/{id}
+```
+
+## Params
+
+| Parameter name | Type | Description |
+| -------------- | ------ | ------------ |
+| `id` | number | The record ID of the user to return |
+
+## Request headers
+
+| Key | Value |
+|---|---|
+| Content-Type | application/json |
+
+## Request body parameters
+
+| Property name | Type | Description |
+| ------------- | ----------- | ----------- |
+| `last_name` | string | The user's last name |
+| `first_name` | string | The user's first name |
+| `email` | string | The user's email address |
+| `id` | number | The user's unique record ID |
+
```

Comment:
Can you change the `id` value?

Can you just specify the property or properties that you want to change and leave the rest of them unchanged by not including them?

Reply:
You cannot change the ID, which I have noted in the parameters section.

I have an alert on lines 9-10 that addresses the need to include properties even if you want them unchanged. With this newest commit I have added a qualifier that ID remains the same regardless.

Reply:
See my other comments above.